tools/swift-demangle/swift-demangle.cpp

```diff
@@ -30,12 +30,16 @@
 
 #include <cstdlib>
 #include <string>
-#if !defined(_MSC_VER) && !defined(__MINGW32__)
+#if !defined(_WIN32) && !defined(__MINGW32__)
 #include <unistd.h>
 #else
 #include <io.h>
 #endif
 
+#include <iostream>
+#include <string>
+#include <stdio.h>
```

Comment:
Not sure. I've cleaned up the headers in the latest commit, but I haven't tested it on multiple platforms, so we'll see if it all compiles :)